docs/pages/kubernetes-access/helm/guides/gcp.mdx

```diff
@@ -233,7 +233,7 @@ spec:
     solvers:
     - selector:
         dnsZones:
-          - "example.com"                                  # Change this
+          - "example.com"   # Change this. This is listed as "DNS name" on the GCP Cloud DNS page.
```

Comment:
This information may be a bit cumbersome to find if you're following the guide. Maybe move the second sentence in the comment above the code snippet and provide a link to the GCP Cloud DNS page. A screenshot of the page may be even clearer.